### PR TITLE
[WC-1999] chore(charts-web): update changelog

### PR DESCRIPTION
**Pull request by iobuhov**
### Pull request type

No code changes (changes to documentation, CI, metadata, etc.)

---

### Description
Add a note about potential breaking changes.

### CODE REVIEW

**Review by gjulivan — DISMISSED** [review]

**Review by samuelreichert — APPROVED** [review]

File: packages/pluggableWidgets/charts-web/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   We fixed an issue where, in 4.0, parameters related to on-click actions were missing. In versions of the widget before 4.0, when you added an on click action to the widget, additional parameters were passed. These parameters are now being passed again.
 
+### Breaking changes
+
+-   This version includes XML changes in some of the widgets, which means that you may notice that after updating your widget, these settings lost their configured value: `Line color` (Line chart), `Marker color` (Line chart), `On click action` (all widgets except Pie chart and Heat map).
```
Comment (r0b1n) [inline packages/pluggableWidgets/charts-web/CHANGELOG.md:15]: _This version includes XML changes in some of the widgets_ - those are internal details, they add no value for our users.

_which means that you may notice_ - let's put it in a more concrete form of stating the fact that there are breaking changes, saying that **if users upgrade** to this version and they had those settings configured they **will** loose the values and they **have to** configure those  settings again.

Comment (r0b1n) [reply packages/pluggableWidgets/charts-web/CHANGELOG.md:15]: BTW, did we fix the issue with back ticks ( ` ) breaking while creating a release? If not, we should not use them.

Comment (iobuhov) [reply packages/pluggableWidgets/charts-web/CHANGELOG.md:15]: I think it should be fixed. I remember fixing it.

**Review by r0b1n — APPROVED** [review]

**Review by takumagott — APPROVED** [review]
LGTM

**Pull request merged**